src/core/lib/experiments/experiments.yaml

```diff
@@ -205,6 +205,15 @@
   expiry: 2024/01/01
   owner: vigneshbabu@google.com
   test_tags: []
+- name: separate_ping_from_keepalive
+  description:
+    Keep a different keepalive timeout (resolution is seeing data after sending a ping)
+    from a ping timeout (resolution is getting a ping ack after sending a ping)
+    The first can be short and determines liveness.
```

Comment:
It took me some time to parse this. Recommend s/first/keepalive\ timeout and s/second/ping\ timeout